spanner/tests/system/test_system.py

```diff
@@ -220,6 +220,12 @@ def _assert_timestamp(self, value, nano_value):
             self.assertEqual(value.microsecond * 1000, nano_value.nanosecond)
 
     def _check_row_data(self, row_data, expected=None):
+        try:
+            row_data = sorted(row_data)
+            expected = sorted(expected)
+        except TypeError:
```

Comment:
@chemelnucfin When will this fail with a `TypeError`?

I'd be more comfortable having the caller be explicit. So you could have a flag `sort=True` and then the callers which are un-sortable would have to change from the default to `sort=False`